README.md

```diff
@@ -1,25 +1,91 @@
-# psibase
+<div align="center">
```

Comment:
I'm not a fan of including raw html content. The whole advantage of markdown is that it's readable as plain text. I'm not sure that it's worth it just to make it center-justified.